src/EnergyPlus/ElectricBaseboardRadiator.cc

```diff
@@ -113,16 +111,9 @@ namespace ElectricBaseboardRadiator {
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine simulates the Electric Baseboard units.
 
-        // REFERENCES:
-        // Water baseboard module
-
-        // Using/Aliasing
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int BaseboardNum; // Index of unit in baseboard array
-        auto &ElecBaseboard = state.dataElectBaseboardRad->ElecBaseboard;
```

Comment:
Yeah, these weren't helpful and down later in the function, once the correct BaseboardNum is found, it's not really worth making a reference.  This is fine.